content/_help/verify-your-identity/how-to-verify-your-identity._en.md

```diff
@@ -65,11 +65,12 @@ Login.gov is a secure, government website that adheres to the highest standards
         * Take an image of the back of your ID
     * You may also need to take a photo of yourself to complete this step.
     * Click “Continue”
-1.  Enter your social security number and click “Continue”
-1.  Verify the information you have submitted so far is correct and click “Continue”
-1.  Enter your primary phone number. This is the last step for identity verification.
-    * If you do not have a U.S. phone number or cannot verify by phone, click on “Verify your address by mail instead.” You will have to wait 5 to 10 days to receive a letter in the mail and then follow the instructions to enter a verification code.
-1.  Once your identity is verified, you need to encrypt your account. Re-enter your Login.gov password. Click “Continue”
-1.  Take note of your personal key and keep it safe. You’ll need to enter your personal key immediately on the next page. Click “Continue”
+3.  Enter your social security number and click “Continue”
+4.  Verify the information you have submitted so far is correct and click “Continue”
+5.  Enter your primary phone number. This is the last step for identity verification.
+    * In some cases, you may be able to select “Verify your address by mail instead.”
+    * If you do not have the option to verify by mail, you must verify by phone to successfully verify your identity.
```

Comment:
Gah, changing the number made this a confusing diff. I changed the two bullet points under 5 (i.e., the third 1. in this set).

The changes makes the diff messy, but it also kind of shows _why_ I think it reads so much better for stuff like this.

Reply:
For me the Chinese translation is where I really needed it: https://github.com/GSA-TTS/identity-site/pull/1307/files/3af5145c1f4e073cfb7bbc345f23ea21480f9c70#diff-ac2b9aa4414232fb76ed121d0c7ef62ba9d65e2376aeef36c9b964dcba7577f8R64-R67

In that case someone had already numbered the file that way, and it's the only reason I felt confident making the translation changes. I decided to extend the pattern to the other files.

If we were English-only (or used Rails-style placeholder strings) I'd be all for keeping the 1. 1. approach.

Reply:
Ah, yeah, I can see that making sense for other languages in particular.